modules/gallvp/helitronscanner/draw/main.nf

```diff
@@ -0,0 +1,70 @@
+process HELITRONSCANNER_DRAW {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/helitronscanner:1.0--hdfd78af_0':
+        'biocontainers/helitronscanner:1.0--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    tuple val(meta2), path(head)
+    tuple val(meta3), path(tail)
+
+    output:
+    tuple val(meta), path("*.draw*")    , emit: draw
```

Comment:
I dont' think we need `*.draw*`. `*.draw` should do the job.

Reply:
This has been resolved as the output file was `.draw.hel.fa` which is now moved to `.draw`